css/styles.css

```diff
@@ -2496,7 +2496,7 @@ ul#menu ul.ssmenu {
 }
 
 .copyright {
-   font-size: 9px;
+   font-size: 9px!important;
```

Comment:
"a, a:link" will have precedence; it was not required before because class was applied on a span element.